packages/website/package.json

```diff
@@ -1,103 +1,94 @@
 {
   "name": "@web3-storage/website",
-  "version": "1.10.0",
+  "version": "2.0.0",
```

Comment:
Ah, got it. Reverted and changed this PR title to `feat!:` format. So I assume this will be the squashed commit title and get incremented by release please as a major version?

Reply:
yes exactly